commands/Administration/welcome.js

```diff
@@ -27,13 +27,44 @@ class Welcome extends Command {
             message.client.emit("guildMemberAdd", message.member);
             return message.channel.send(message.language.get("WELCOME_TEST_SUCCESS"));
         }
+        if(args[0] === "color" && data.settings.plugins.welcome.enabled && data.settings.plugins.welcome.withImage){
+            //Color ?
+            if(args[1]){
+                // Hexadecimal value
+                if(args[1].startsWith("#")){
+                    let color = args[1],
+                    hexacolor = color.replace(/[^a-z0-9]/gi, "");
+                    if(hexacolor.length === 6){
+                        data.settings.plugins.welcome.color = "#"+hexacolor.toUpperCase();
+                        data.settings.markModified("plugins.welcome");
+                        data.settings.save();
+                        return message.channel.send(message.language.get("WELCOME_SUCCESS_COLOR", data.settings.plugins.welcome.color, data.settings.prefix));
+                    } else {
+                        return message.channel.send(message.language.get("WELCOME_ERR_COLOR"));
+                    }
+                }
+                // Defaults values
+                if(args[1] === "white" || args[1] === "orange" || args[1] === "red" || args[1] === "blue" || args[1] === "green" || args[1] === "purple"){
```

Comment:
It would be necessary to add a `toLowerCase()` here
`if(args[1].toLowerCase() === "white" &&`...